apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     repo: "CodeForAfrica/ui",
     proxy_url: "http://localhost:8081/api/v1", // Set proxy to work on local repo
   },
-  media_folder: "public/images",
+  media_folder: "apps/codeforafrica/public/images",
```

Comment:
We can just declare `const APP_DIRECTORY` or env var `NEXT_PUBLIC_APP_DIRECTORY` (or both, i.e. declare `APP_DIRECTORY` initialized from `NEXT_PUBLIC_APP_DIRECTORY` env var) and reuse it rather than repeating `apps/codeforafrica`.